packages/wdio-types/src/Capabilities.ts

```diff
@@ -89,10 +90,36 @@ export interface MultiRemoteCapabilities {
 }
 
 export type RemoteCapability = DesiredCapabilities | W3CCapabilities | MultiRemoteCapabilities;
-export interface Worker extends Omit<TestrunnerOptions, 'capabilities'> {
+
+export interface Job {
+    caps: DesiredCapabilities | W3CCapabilities | MultiRemoteCapabilities
+    specs: string[],
+    hasTests: boolean
+}
+
+export type WorkerMessageArgs = Omit<Job, 'caps' | 'specs' | 'hasTests'>;
+
+export interface WorkerRunPayload {
+    cid: string;
+    configFile: string;
+    caps: RemoteCapability;
+    specs: string[];
+    execArgv: string[];
+    retries: number;
+}
+
+export interface Worker extends Omit<TestrunnerOptions, 'capabilities' | 'specs'>, EventEmitter {
     capabilities: RemoteCapability
+    caps: RemoteCapability
+    cid: string;
+    isBusy?: boolean
+    postMessage: (command: string, args: WorkerMessageArgs) => void
```

Comment:
@christian-bromann Re-reviewing my own code, I relocated the types that I did to accommodate this change right here. Without relocating these types to `@wdio/types`:

- `@wdio/local-runner` would need to add `@wdio/cli` as a devDependency to access `Job` (if we were to restore `Job`'s original source file location)
- `@wdio/types` would then need to add `@wdio/local-runner` as a devDependency to access `WorkerMessageArgs` (as seen here)

(Meanwhile, the `Worker` interface is imported by both `@wdio/cli` and `@wdio/local-runner`.)

Reply:
I am open to suggestion, of course, but anyway, this seems to be the root pain point in terms of type definition location.

Reply:
I was not suggesting to move them back to their original place. I suggested to move them into a neutral place in the `@wdio/types` package, e.g. `packages/wdio-types/src/index.ts`.

Reply:
Ah, got it. That sounds great, will do.

Reply:
@christian-bromann Thanks again for the guidance; please take another look at the type definition locations and imports now that I've pushed 19abf23.

Reply:
Can you also move `Worker` and `WorkerPool` types to `packages/wdio-types/src/index.ts`? Same reason here: they are not very related to capabilities.